crates/zed/src/languages/csharp/highlights.scm

```diff
@@ -0,0 +1,469 @@
+(identifier) @variable
+
+((identifier) @keyword
+  (#eq? @keyword "value")
+  (#has-ancestor? @keyword accessor_declaration))
+
+((identifier) @variable.builtin
+  (#eq? @variable.builtin "_"))
+
+(method_declaration
+  name: (identifier) @function.method)
```

Comment:
We don't use `function.method`, and several other highlights in this file. for the list of annotations we do support, see: https://github.com/zed-industries/zed/blob/main/crates/theme/src/styles/syntax.rs#L16-L68

Reply:
Thanks! I'll update these, the other highlights, and add the `outline.scm`. It looks like the other highlights in the repo do use `function.method` though. Maybe those need updated too?

Reply:
I made these updates and added reference screenshots comparing to VS Code. Let me know if you need anything else.